questions/javascript-hoisting-scope.md

```diff
@@ -0,0 +1,22 @@
+### In Javascript, Hoisting happens to functions and variables declared with `var`. Where can they be scoped?
+
+#### Answer
+
+Either globally or in a function.
+
+Unlike `let` or `const` they are not block-scoped and can only exist in either one of them because they are assigned a pointer before any evaluation of the code.
```

Comment:
We could explain this further :smiley:

Reply:
probably. Finding the right balance though without repetition is the issue...